### ISSUE TITLE
[Squeak 5.3] Installation asks for author initials

### ISSUE THREAD

**Issue by LinqLover** [hpi-swa/Squot#383]
While installing Squot in a fresh Squeak 5.3 image, I hit an unexpected "enter author initials" question:

![image](https://user-images.githubusercontent.com/38782922/189545169-0cdb8ef9-68da-4e98-bb54-9ca6a1a4ba40.png)

In other scenarios, this might break CIs. Why can't we use `-override` extension methods here?

**Comment by j4yk** [comment]
Because I did not want to override the existing methods and functionality if it is already present.

Now that Squeak 6.0 is here I could reconsider.

**Comment by j4yk** [comment]
Since this is in FileSystem-Git, not in Squot, I would have to create compatibility packages there in the first place. But wait, maybe the SquotCompatibility-SqueakCommon package is actually mislabelled now, since I certainly did not rely on Pharo methods in the "new" code (as compared to the "old" code in FileSystem-Git and FileSystem)!

**Comment by j4yk** [comment]
Indeed, only FileSystem-Git code uses things from SquotCompatibility-SqueakCommon. With the old name Pharo-Compatbility it did not matter which code used it. :-) So I could rename that package.

But for the stub methods for read-only objects I would anyway need another compat package because the methods are available in Squeak 6.0, and the SquotCompatibility-Squeak50 has indeed methods used by Squot, not by FileSystem-Git.
